private/numerics/functions/bessel.rkt

```diff
@@ -0,0 +1,374 @@
+		     -.10497327982345548331260e+1
+		     -.93525953294031893049e-2)
+                    (poly-by-coeffs->value
+                     z²
+                     +.2400674237117267479318819e+4
+		     +.2971983745208491990065486e+4
+		     +.921566975526530895082307e+3
+		     +.74428389741411178824152e+2
+		     +.1e1))])
+        (* (sqrt (/ 2/π |x|))
+           (+ (* (sin |x|-π/4) p₀)
+              (* z (cos |x|-π/4) q₀))))))
+
+(define (bessy₁ x)
+  (define |x| (magnitude x))
+  (if (< |x| 8.0)
+      (let* ([x² (* x x)]
+             [r₀ (poly-by-coeffs->value
+                  x²
+                  -.2493247725431151099221863985e+8  ;p00
+                  +.678814979608784027013965029e+7   ;p01
+                  -.3418758685257648961162234201e+6  ;p02
+                  +.731876663732252704384675659e+4   ;p03
+                  -.8477929772037826827977473917e+2  ;p04
+                  +.5973109455969101918912869725e+0  ;p05
+                  -.2723714918114737334647812804e-2  ;p06
+                  +.8253358475754237969740284405e-5  ;p07
+                  -.1645797675540583390670878295e-7  ;p08
+                  +.2013974632712911344982964612e-10 ;p09
+                  -.118503924369772697029706524e-13  ;p10
+                  )]
+             [s₀ (poly-by-coeffs->value
+                  x²
+                  +.1271694748306711445338693265e+9  ;q00
+                  +.6291245810783959009675422035e+6  ;q01
+                  +.1241151964683171602676430057e+4  ;q02
+                  +1.0                               ;q03
+                  )])
+        (+ (/ (* x r₀) s₀)
+           (* 2/π
+              (- (* (bessj₁ x) (log x))
+                 (/ 1.0 x)))))
+      (let* ([z (/ 8.0 |x|)]
+             [z² (* z z)]
+             [|x|-3π/4 (- |x| 3π/4)]
+             [p₁ (/ (poly-by-coeffs->value
+                     z²
+                     +.1290918471896188077350689e+5
+		     +.1309042051103506486292571e+5
+		     +.313275295635506951011069e+4
+		     +.17431379748379024599685e+3
+		     +.122850537643590432633e+1)
+                    (poly-by-coeffs->value
+                     z²
+                     +.1290918471896187879332737e+5
+		     +.1306678308784402036110575e+5
+		     +.310928141677002883350924e+4
+		     +.16904721775008609992033e+3
+		     +.1e+1))]
+             [q₁ (/ (poly-by-coeffs->value
+                     z²
+                     +.14465282874995208675225e+3
+		     +.1744291689092425885102e+3
+		     +.5173653281836591636536e+2
+		     +.379944537969806734901e+1
+		     +.36363466476034710809e-1)
+                    (poly-by-coeffs->value
+                     z²
+                     +.308592701333231723110639e+4
+		     +.373434010601630179517765e+4
+		     +.11191098527047487025919e+4
+		     +.8522392064341340397334e+2
+		     +.1e+1))])
+        (* (sqrt (/ 2/π |x|))
+           (+ (* (sin |x|-3π/4) p₁)
+              (* z (cos |x|-3π/4) q₁))))))
+
+(define (bessy n x)
```

Comment:
There's something wrong with all three `bessy` functions. Each generates an empty graph. The original looks like:
![bessy-orig](https://cloud.githubusercontent.com/assets/1731829/8680264/2355f840-2a2e-11e5-8dc0-58d948c27328.png)